### ISSUE TITLE
Next step is unclear after pressing "Report Issue" for perf problem

### ISSUE THREAD

**Issue by debonte** [microsoft/pylance-release#5833]
## Environment data

-   Language Server version: 2024.4.105

While [testing](https://github.com/microsoft/pyrx/issues/5104) the new `python.analysis.perfReporting` feature, I pressed the "Report Issue" button and was expecting that it was going to either collect all the info it needed about the perf problem or tell me how to do it. Looking at the GitHub issue template, it seems to have done neither of those things.

I think we should update the template so there is a very obvious pointer to instructions on collecting profiling data. I'd suggest adding it as part of the comment below -- maybe there should be a separate link to "Filing a perf issue". 

```html
<!--
Read the guidelines for 'Filing an issue'.

'https://aka.ms/pylance-filing-an-issue'
-->
```

Additionally, maybe there should be a separate issue template for perf bugs, and this "Report Issue" command should use that perf-centered template?

**Comment by heejaechang** [comment]
good idea. for now, intension was helping users to create an issue to us (even if it doesn't have much info in it) so we have open channel to communicate with users. once we have the channel opened, details could be obtained from users.

**Comment by rchiodo** [comment]
Not sure we have this button anymore? Closing
